sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/MetricsAdvisorAdministrationClientLiveTests.cs

```diff
@@ -166,16 +169,30 @@ public async Task HookOperations()
         {
             var adminClient = GetMetricsAdvisorAdministrationClient();
 
-            AlertingHook createdHook = await adminClient.CreateHookAsync(new EmailHook(Recording.GenerateAlphaNumericId("test"), new List<string> { "foo@contoso.com" })).ConfigureAwait(false);
-            AlertingHook getHook = await adminClient.GetHookAsync(createdHook.Id).ConfigureAwait(false);
-            List<AlertingHook> hooks = await adminClient.GetHooksAsync(new GetHooksOptions { HookName = getHook.Name }).ToEnumerableAsync().ConfigureAwait(false);
+            AlertingHook createdEmailHook = await adminClient.CreateHookAsync(new EmailHook(Recording.GenerateAlphaNumericId("test"), new List<string> { "foo@contoso.com" }) { Description = $"{nameof(EmailHook)} desscription" }).ConfigureAwait(false);
```

Comment:
nit: `desscription` => `description`

For 1 sec I though the updated description would be used to fix the typo.